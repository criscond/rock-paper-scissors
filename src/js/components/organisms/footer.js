import { copyright } from '../../data';
import { createElement } from '../services';

const footer = () => {
  const element = createElement({ htmlTag: 'footer' });
  const paragraph = createElement({ htmlTag: 'p' });
  const year = new Date().getFullYear();
  paragraph.innerHTML = `Â© ${year} ${copyright}`;
  element.appendChild(paragraph);

  return element;
}

export default footer;
