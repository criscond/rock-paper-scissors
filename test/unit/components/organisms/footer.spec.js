import footer from '../../../../src/js/components/organisms/footer';
describe('Footer', () => {

	it('should create the footer', () => {
		expect(footer()).toMatchSnapshot();
	});
});
