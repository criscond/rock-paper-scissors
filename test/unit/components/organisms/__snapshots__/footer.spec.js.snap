// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should create the footer 1`] = `
<footer>
  <p>
    Â© 2022 CrisCond
  </p>
</footer>
`;
